test/basic.html

```diff
@@ -206,6 +206,34 @@
           }, 1);
         });
 
+        test('arrow keys cause iron-activate events', function(done) {
+          var g = fixture('WithSelection');
+
+          g.items[0].focus();
+
+          var i = 0;
```

Comment:
I thiiiiink you might need to wrap everything that's not in the listener in a `Polymer.Base.async` because everything is ready a tick later (due to `observeNodes` in the selectable behaviour)